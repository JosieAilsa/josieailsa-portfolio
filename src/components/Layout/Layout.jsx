import Nav from "../../containers/Nav/Nav";
const Layout = ({ children }) => {
  return (
    <>
      <Nav />
      {children}
    </>
  );
};

export default Layout;
