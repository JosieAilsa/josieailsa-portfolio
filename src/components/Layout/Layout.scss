@use "../../assets/sass/abstracts/variables" as * ;
@use "../../assets/sass/abstracts/mixins" as * ;
@use "../../assets/sass/base/resets" as * ;
@use "../../assets/sass/base/typography" as * ;