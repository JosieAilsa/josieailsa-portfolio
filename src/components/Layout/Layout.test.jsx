import { customRender } from "../../utils/testutils";
import Layout from "./Layout";

test("Home page should render", () => {
    const { container } = customRender(<Layout/>);
    expect(container).toMatchSnapshot();
  });
  