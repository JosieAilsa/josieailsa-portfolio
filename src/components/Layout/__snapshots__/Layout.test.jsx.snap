// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should render 1`] = `
<div>
  <header>
    <nav
      class="navbar"
    >
      <img
        alt="logo"
        class="navbar__logo"
        src="jnlogo-02.svg"
      />
      <ul
        class="navbar__links"
      >
        <li
          class="navbar__link"
        >
          projects
        </li>
        <li
          class="navbar__link"
        >
          skills
        </li>
        <li
          class="navbar__link"
        >
          contact
        </li>
      </ul>
      <img
        alt="nav-menu"
        class="navbar__hamburger"
        src="hamburger.svg"
      />
    </nav>
  </header>
</div>
`;
